collector/builder-config.yaml

```
@@ -114,6 +114,7 @@ connectors:
 providers:
   - gomod: go.opentelemetry.io/collector/confmap/provider/envprovider v0.126.0
   - gomod: go.opentelemetry.io/collector/confmap/provider/odigosk8scmprovider v0.126.0
+  - gomod: go.opentelemetry.io/collector/confmap/provider/fileprovider v1.32.0
```

please keep the consistency in the versions. different dependency version can cause super hard to handle issues
```suggestion
  - gomod: go.opentelemetry.io/collector/confmap/provider/fileprovider v0.126.0
```